app/src/main/java/org/fossasia/openevent/general/event/EventsFragment.kt

```diff
@@ -143,8 +143,8 @@ class EventsFragment : Fragment(), BottomIconDoubleClick {
 
         rootView.notification.isVisible = eventsViewModel.isLoggedIn()
         rootView.notificationToolbar.isVisible = eventsViewModel.isLoggedIn()
-        rootView.newNotificationDot.isVisible = eventsViewModel.isLoggedIn()
-        rootView.newNotificationDotToolbar.isVisible = eventsViewModel.isLoggedIn()
+//        rootView.newNotificationDot.isVisible = eventsViewModel.isLoggedIn()
+//        rootView.newNotificationDotToolbar.isVisible = eventsViewModel.isLoggedIn()
```

Comment:
revert this. 
How it can be a solution to the issue?

Reply:
Okay, I will simply delete these lines instead of commenting them.